config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
 	namespace :api do
 	  namespace :v1 do
-	      resources :faqs, only: [:index]
+	      resources :faqs,       only: [:index]
```

Comment:
fix the indentation of these lines. All the resources should be at the same indentation level